app/views/nsm/steps/view_claim/_adjustments.html.erb

```diff
@@ -1,13 +1,13 @@
 <div class="govuk-tabs" data-module="govuk-tabs">
   <ul class="govuk-tabs__list">
     <li class="govuk-tabs__list-item <%= 'govuk-tabs__list-item--selected' if @scope == :work_items %>">
-      <%= link_to t(".#{prefix}work_items"), polymorphic_path([@section, :work_items, :nsm, :steps_view, claim], anchor: 'claim_nav') %>
+      <%= link_to t(".#{prefix}work_items"), send("#{@section}_work_items_nsm_steps_view_claim_path", claim, anchor: 'claim_nav') %>
```

Comment:
What is `send` here?

Reply:
`send` is how you call a method on a ruby object if all you have is the name's method as a string. `a.send("foo")` is broadly the same as `a.foo`. In this case we can no longer use `polymorphic_path` because that uses some magic that hinges on taking the `claim` and, because it's a `Claim` object, inferring that the required path helper method has `_claim_` in its name. But Now `claim` is a `AppStore::V1::Nsm::Claim`, that same magic tries to find a path helper method with `_app_store_v1_nsm_claim_` in its name, and there obviously is no such path helper. So we have to do a different sort of metaprogramming, in this case constructing the helper method name ourselves as a string and invoking it with `send`